python/lsst/validate/drp/util.py

```diff
@@ -150,29 +174,26 @@ def loadDataIdsAndParameters(configFile):
     ----------
     configFile : str
         YAML file that stores visit, filter, ccd,
+        and additional configuration parameters such as
         brightSnr, medianAstromscatterRef, medianPhotoscatterRef, matchRef
 
     Returns
     -------
-    dict, float, float, float
-        dataIds, brightSnr, medianRef, matchRef
+    pipeBase.Struct
+        with attributes of
+        dataIds - dict
+        and configuration parameters
     """
-    stream = open(configFile, mode='r')
-    data = yaml.load(stream)
+    parameters = loadParameters(configFile).getDict()
 
-    ccdKeyName = getCcdKeyName(data)
+    ccdKeyName = getCcdKeyName(parameters)
     try:
-        visitDataIds = constructDataIds(data['filter'], data['visits'],
-                                        data[ccdKeyName], ccdKeyName)
-    except KeyError as ke:
-        visitDataIds = []
+        dataIds = constructDataIds(parameters['filter'], parameters['visits'],
+                                   parameters[ccdKeyName], ccdKeyName)
+    except KeyError:
+        dataIds = []
```

Comment:
Is the meaning here that if a filter or CCD are missing for a visit (in any visit) then no data IDs are returned?